### PR TITLE
Dev client app

### PR DESCRIPTION
**Pull request by alexeses**
Hello team,

I'm pleased to share the Pull Request featuring the fully functional Android application. EatUP is almost ready to be presented, and we're making great progress.

Best regards,
Alex.

### CODE REVIEW

File: app/EatUp_Client/app/proguard-rules.pro
Comment (Jkutkut) [inline app/EatUp_Client/app/proguard-rules.pro:1]: Nice! I didn't know this was a thing!

File: app/EatUp_Client/app/src/androidTest/java/com/github/eatup_client/ExampleInstrumentedTest.java
Comment (Jkutkut) [inline app/EatUp_Client/app/src/androidTest/java/com/github/eatup_client/ExampleInstrumentedTest.java:1]: Nice!

File: app/EatUp_Client/app/src/main/java/com/github/eatup_client/api/ProductApiService.java [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+/**
+ * Service class for making API calls related to products.
+ */
+public class ProductApiService {
+
+    private static final String BASE_URL = "http://159.69.216.101/api/v1/";
```
Comment (Jkutkut) [inline app/EatUp_Client/app/src/main/java/com/github/eatup_client/api/ProductApiService.java:43]: This constant will allow an easy setup for dynamic urls in the future. Great work!

File: app/EatUp_Client/app/src/main/java/com/github/eatup_client/model/Allergy.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @SerializedName("id")
+    @Expose
+    private String id;
+    @SerializedName("name")
+    @Expose
+    private String name;
+    @SerializedName("img_id")
+    @Expose
+    private String img_id;
```
Comment (Jkutkut) [inline app/EatUp_Client/app/src/main/java/com/github/eatup_client/model/Allergy.java:18]: Caution! This is no longer supported in server v1.0.0. This is not throwing any errors?

**Review by Jkutkut — APPROVED** [review]
Nice work! Check if the allergies throws any error with the current version (already deployed) and you have my OK

**Review by mariaduva — APPROVED** [review]

**Pull request merged**